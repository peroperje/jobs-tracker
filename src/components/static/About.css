.fade-appear {
    opacity: 0.01;
    z-index: 1;
}

.fade-appear-active {
    opacity: 1;
    transition: opacity 600ms ease-in;
}

.fade-leave {
    opacity: 1;
}

.fade-leave-active {
    opacity: 0.01;
    transition: opacity 600ms ease-in;

}