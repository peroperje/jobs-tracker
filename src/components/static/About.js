import React from 'react';


import './About.css';
/**
 * @function About
 * @param {Object} props property object
 * @constructor
 */
function About(props) {
  return (

        <h1 style={{textAlign: 'center'}}>About page</h1>

  );
}

export default About;
