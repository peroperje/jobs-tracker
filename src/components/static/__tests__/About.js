import React from 'react';
import renderer from 'react-test-renderer';

import About from '../About';

describe('About component', () => {

  it('About Snapshot ', () => {
    const tree = renderer.create(<About/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
