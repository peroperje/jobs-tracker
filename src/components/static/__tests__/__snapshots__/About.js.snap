// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component About Snapshot  1`] = `
<h1
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  About page
</h1>
`;
